README.md

```diff
@@ -39,6 +39,39 @@ to get started is with the official Docker image, sensu/sensu.
 
 See the [installation documentation](https://docs.sensu.io/sensu-go/latest/installation/install-sensu/) to get started.
 
+### Building from source
+
+The various components of Sensu Go can be manually built from this repository.
+You will first need [Go](https://golang.org/doc/install#install) installed.
+Then, you should clone this repository **outside** of the GOPATH since Sensu Go
+uses [Go Modules](https://github.com/golang/go/wiki/Modules).
+
+To compile and then run Sensu Go within a single step:
+```
+$ go run ./cmd/sensu-agent/main.go
+$ go run ./cmd/sensu-backend/main.go
+$ go run ./cmd/sensuctl/main.go
```

Comment:
```suggestion
$ go run ./cmd/sensu-agent
$ go run ./cmd/sensu-backend
$ go run ./cmd/sensuctl
```

Reply:
This makes the instructions less fragile, since if we add more files or change the name of main.go, the command will still work.

Reply:
Ha, I don’t know why but I always thought go run didn’t supported directories. TIL